### DISCUSSION TITLE
Change the text size

### DISCUSSION THREAD

**Discussion by mikrobFIZ** [primefaces/community#3003] [Figma UI Kit]
Where is the best position to change the font size? In the variables or in the Token Studio? I can't find an entry for the text size in the Token Studio.

**Comment by onursenture** [comment]
Hello, we recommend making changes to tokens using Token Studio. The font size tokens are located under the **app** set. It would also be beneficial for you to check the Base font size section in the Settings tab of Tokens Studio.
